app/Models/BeatmapsetWatch.php

```diff
@@ -65,6 +65,11 @@ public function scopeRead($query)
         });
     }
 
+    public function scopeUnread($query)
+    {
+        $query->whereColumn('last_read', '<', 'last_notified');
```

Comment:
Nothing currently updates `last_notified` so the result is wrong.

Reply:
huh... I just flipped scopeRead/isRead expecting it to work, how does `isRead()` have the right result on live site?

Reply:
It's not currently used anywhere.

Reply:
er, sorry, it is used somewhere. But it's currently wrong as the original thing that updates the column has been replaced with something else.